include/param/param.h

```diff
@@ -181,7 +184,7 @@ typedef struct param_s {
 	; /* Catch const param defines */ \
 	uint32_t _timestamp_##_name = 0; \
 	__attribute__((section("param"))) \
-	__attribute__((aligned(1))) \
+	__attribute__((aligned(8))) \
```

Comment:
Why do we actually need to align the param_t variable?

Reply:
In this case we actually do not need to do so. This alignment has been a part of a debugging process, in which I found that it was the entire list of param_t objects in FLASH (linker script file .ld) which needed to be aligned to 8 bytes (64-bit). This can be removed.